### PR TITLE
Coord format, Pacific/Oceanic sections, Best Practices page, SY TCU

### PR DESCRIPTION
**Pull request by mattkelly4**

### CODE REVIEW

File: docs/aerodromes/Alice.md [hunk 1]
```diff
@@ -29,16 +29,16 @@ For simulation purposes, visual separation is assumed to exist below the cloud b
 Departures from YBAS in to ASP Class C will be coordinated when ready for departure.
 
 !!! example
-    **AS ADC** -> **ASP**: "Next, QFA797"  
-    **ASP** -> **AS ADC**: "QFA797"
+    <span class="hotline">**AS ADC** -> **ASP**</span>: "Next, QFA797"
```
Comment (alexd-332) [inline docs/aerodromes/Alice.md:32]: did we decide we were going to go with coldlines for initial calls? some others below may need to be fixed

File: docs/aerodromes/Archerfield.md [hunk 2]
```diff
@@ -121,8 +122,8 @@ Archerfield Tower will not alter an aircrafts tracking until the aircraft is wit
 #### Visual Tracking
 
 !!! example
-    **BN TCU** -> **AF ADC**: "Estimate, UJE via TVT time 02."  
-    **AF ADC** -> **BN TCU**: "UJE"
+    <span class="coldline">**BN TCU** -> **AF ADC**</span>: "Estimate, UJE via TVT time 02."
```
Comment (alexd-332) [inline docs/aerodromes/Archerfield.md:125]: coord example needs to be changed to standard heads-up format (via TVT, UJE), as we try to get rid of the obsolete estimates, routings, etc

File: docs/aerodromes/Bankstown.md [hunk 3]
```diff
@@ -150,19 +151,19 @@ SY TCU will coordinate with BK ADC for incoming IFR arrivals on either Visual Tr
...
-    **SY TCU** -> **BK ADC**: "Estimated approach time, UJN via RNP at time 59"  
-    **BK ADC** -> **SY TCU**: "Via RNP, UJN"   
+    <span class="coldline">**SY TCU** -> **BK ADC**</span>: "Estimated approach time, UJN via RNP at time 59"  
+    <span class="coldline">**BK ADC** -> **SY TCU**</span>: "Via RNP, UJN"   
 
     *BK Tower must clear aircraft for final, before SY TCU issues approach clearance.*  
-    **SY TCU** -> **BK ADC**: "Request clearance for final, UJN"  
-    **BK ADC** -> **SY TCU**: "UJN cleared RNP 11C"  
-    **SY TCU** -> **BK ADC**: "Cleared RNP 11C, UJN"  
+    <span class="coldline">**SY TCU** -> **BK ADC**</span>: "Request clearance for final, UJN"
```
Comment (alexd-332) [inline docs/aerodromes/Bankstown.md:164]: hotline? others may need to be changed

File: docs/oceanic/index.md [hunk 4]
```diff
@@ -50,6 +50,30 @@ NTTT - Tahiti Oceanic
 
     Honiara (AGGG) and Nauru (ANAU) FIR ATS are provided by Brisbane Oceanic (Tasman).  
 
+## Procedures
+
+### Transition Levels
```
Comment (alexd-332) [inline docs/oceanic/index.md:55]: Would benefit from either a link, or additional table, referencing AIP ENR 1.7 2.5.2, ie QNHs below 1013 precluding cruising at certain levels. I assume the rules used in australia apply the same to other FIRs as well

**Review by alexd-332 — CHANGES_REQUESTED** [review]
also there's a few merge conflicts by the look of it :'( maybe needs some manual work

File: docs/aerodromes/Alice.md (hunk 1 above)
Comment (mattkelly4) [reply docs/aerodromes/Alice.md:32]: It's a bit messy trying to build a standard, but I went with hot for all next calls given that they're a quick pop up sort of thing (and the FDR activation replaces the taxi call)

File: docs/aerodromes/Bankstown.md (hunk 3 above)
Comment (mattkelly4) [reply docs/aerodromes/Bankstown.md:164]: Fixed

File: docs/oceanic/index.md (hunk 4 above)
Comment (mattkelly4) [reply docs/oceanic/index.md:55]: Added note

File: docs/aerodromes/Archerfield.md (hunk 2 above)
Comment (mattkelly4) [reply docs/aerodromes/Archerfield.md:124]: Discussion for the team I think.  I've updated the metro Ds to follow this Estimate format for now, might be worth revisiting.

**Review by alexd-332 — DISMISSED** [review]

**Review by tylerthetiletiler — APPROVED** [review]

**Review by tylerthetiletiler — DISMISSED** [review]

**Review by alexd-332 — APPROVED** [review]

**Review by alexd-332 — APPROVED** [review]

**Review by LeGiraffe — APPROVED** [review]

**Review by alexd-332 — APPROVED** [review]

**Review by alexd-332 — APPROVED** [review]

**Review by alexd-332 — APPROVED** [review]

**Review by LeGiraffe — DISMISSED** [review]

File: docs/oceanic/phraseology.md [hunk 5]
```diff
@@ -1,43 +1,41 @@
...
-## Phraseology
-
-### HF Radio
+## HF Radio
 
 HF radio is used as the primary long-range radio communications medium. The nature of HF radio makes it highly vulnerable to atmospheric distortion and noise, and so radiotelephony (R/T) procedures on HF tend to be more formal in order to maximise clarity.
 
 !!! Example
-    **UAL873** -> **ZAK_W_FSS**: "San Francisco Radio, San Francisco Radio, UAL873 on 122.5"  
-    **ZAK_W_FSS** -> **UAL873**: "UAL873, San Francisco Radio, go-ahead"
```
Comment (chrisg06) [inline docs/oceanic/phraseology.md:15]: Can these callsigns be altered to reflect the updated procedures in ZAK? They control the whole thing now the majority of the time, logon code ZAK_FSS.

Comment (mattkelly4) [reply docs/oceanic/phraseology.md:15]: Fixed

**Review by LeGiraffe — APPROVED** [review]

**Review by alexd-332 — APPROVED** [review]

**Review by zachbf — APPROVED** [review]

**Pull request merged**